tests/Feature/DashboardPageTest.php

```diff
@@ -86,4 +86,23 @@ public function itShowsTheLoggedInUserDetails()
         ;
     }
 
-}
\ No newline at end of file
+    /** @test */
+    public function itShowsThePrintButtonWithReleventTextForPrintPref()
+    {
+        // Set centre print_pref to 'collection'.
+        $this->centre->print_pref = 'collection';
+        $this->centre->save();
+        $this->actingAs($this->user->fresh())
```

Comment:
Why fresh? so we can reload Auth::user()->centre?

Reply:
Yeah - if not fresh - we get the old centre.